apps/retrospective/services/sprint_sync.go

```diff
@@ -336,29 +340,74 @@ func (service SprintService) AssignPoints(sprintID string, sprintTaskID *string)
 	return nil
 }
 
+// GetTimeTrackerData ...
+func (service SprintService) GetTimeTrackerData(sprint retroModels.Sprint, taskProviderConfig []byte) (
+	[]string,
+	map[uint][]timeTrackerSerializers.TimeLog,
+	error) {
+
+	var timeTrackerTaskKeys []string
+	var timeLogs []timeTrackerSerializers.TimeLog
+	sprintMemberTimeLogs := map[uint][]timeTrackerSerializers.TimeLog{}
+	var err error
+
+	if sprint.Retrospective.TimeProviderName == constants.JIRA {
+		timeTrackerTaskKeys, timeLogs, err = service.GetSprintMemberTimeTrackerData(taskProviderConfig, sprint)
+		if err != nil {
+			utils.LogToSentry(err)
+			service.SetSyncFailed(sprint.ID)
+			return nil, nil, err
+		}
+		memberEmailTimeLogs := make(map[string][]timeTrackerSerializers.TimeLog)
+		for _, timeLog := range timeLogs {
+			memberEmailTimeLogs[timeLog.Email] = append(memberEmailTimeLogs[timeLog.Email], timeLog)
```

Comment:
Does it work fine with Maps? Won't the first parameter to append fail if an email is not found.

Reply:
yes, it will work fine.